platforms/quorum/configuration/roles/helm_component/templates/tessera.tpl

```diff
@@ -0,0 +1,81 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: {{ component_name }}
+  namespace: {{ component_ns }}
+  annotations:
+    flux.weave.works/automated: "false"
+spec:
+  releaseName: {{ component_name }}
+  chart:
+    git: {{ git_url }}
+    ref: {{ git_branch }}
+    path: {{ charts_dir }}/node_tessera  
+  values:
+    replicaCount: 1
+    metadata:
+      namespace: {{ component_ns }}
+      labels:
+    images:
+      node: quorumengineering/quorum:{{ network.version }}
+      alpineutils: adopblockchaincloud0502.azurecr.io/alpine-utils:1.0
```

Comment:
previous comment applied to all the items here. even alpineutils should use the docker url from network.yaml
as well has tessera version

Reply:
Sure, I have done it now.